packages/core-contracts/contracts/utils/ContractUtil.sol

```diff
@@ -11,4 +11,26 @@ contract ContractUtil {
 
         return size > 0;
     }
+
+    function _toString(uint256 value) internal pure returns (string memory) {
```

Comment:
this was ported from OZ's utils for uint256

Reply:
this is also safe since it is from OZ but perhaps we can remove all this code overhead

Reply:
This is used in the ERC721Metadata